src/hooks/sdk/docs/useDevicePermissionStatus.stories.mdx

```diff
@@ -4,6 +4,8 @@
 
 The `useDevicePermissionStatus` hook returns the device permission status from the enum below.
 
+This hook cannot read the device permissions from the browser due to browsers' lack of support of such a querying function. Hence, the `permission` will be always set to `UNSET` when the application starts.
```

Comment:
Do some browsers have the querying function and some don't? If there is no API at all we can change the support sentence to something like, "Currently, there is no browser API to query the device permissions status, hence, `useDevicePermissionStatus` hook is always initialized with `UNSET` permission status."

If there is lack of support though we may link it here.

Reply:
@devalevenkatesh Yes, there is an experimental API called [`permisison.query`](https://developer.mozilla.org/en-US/docs/Web/API/Permissions/query). For camera, and mic query, only chrome supports it now, Firefox and safari don't.

Reply:
Cool, thanks for the API.